<launch>
<node name="rmp_distance_node" pkg="cpt_ros" type="rmp_distance_node" output="screen">
    <param name="zero_angle" value="0.4"/>
   <param name="zero_x" value="0.0"/>
   <param name="zero_y" value="0.0"/>
   <param name="zero_z" value="0.707"/>
   <param name="off_path" value="/home/mpantic/Desktop/rss/surfaces/half_sphere2.off" />
   </node>
<!--
 <node name="rmp_coordinate_node" pkg="cpt_ros" type="rmp_coordinate_node" output="screen">
    <param name="zero_angle" value="0.79"/>
   <param name="zero_x" value="0.0"/>
   <param name="zero_y" value="0.0"/>
   <param name="zero_z" value="0.0"/>
   <param name="off_path" value="/home/mpantic/Desktop/rss/surfaces/flat.off" />
   </node>-->
<!--
   <node name="rmp_coordinate_node" pkg="cpt_ros" type="rmp_coordinate_node" output="screen">
    <param name="zero_angle" value="0.59"/>
   <param name="zero_x" value="0.526"/>
   <param name="zero_y" value="0.7237"/>
   <param name="zero_z" value="0.2319"/>
   <param name="off_path" value="/home/mpantic/Desktop/rss/surfaces/crazy.off" />
</node>-->
<!--
<node name="rmp_coordinate_node" pkg="cpt_ros" type="rmp_coordinate_node" output="screen">
    <param name="zero_angle" value="0.26"/>
   <param name="zero_x" value="0.45"/>
   <param name="zero_y" value="0.9"/>
   <param name="zero_z" value="0.64"/>
   <param name="off_path" value="/home/mpantic/Desktop/rss/surfaces/doublycurved.off" />
</node>
-->
    <node name="spacenav_node" pkg="spacenav_node" type="spacenav_node">
    </node>
</launch>
